modules/luci-mod-status/htdocs/luci-static/resources/view/status/include/20_mount.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+'require fs';
+'require rpc';
+
+var callMountPoints;
+
+callMountPoints = rpc.declare({
+	object: 'luci',
+	method: 'getMountPoints',
+	expect: { result: [] }
+});
+
+return L.Class.extend({
+	title: _('Mount Points'),
+	
+	load: function() {
+		return Promise.all([
+			callMountPoints()
+		]);
+	},
+
+	render: function(data) {
+
+		var mounts = data[0] || [];
+
+		if (!L.hasSystemFeature('block'))
+			return null;
+
+		var table = E('table', { 'class': 'table' }, [
+			E('tr', { 'class': 'tr table-titles' }, [
+				E('th', { 'class': 'th' }, _('Device')),
+				E('th', { 'class': 'th' }, _('Mount Point')),
+				E('th', { 'class': 'th' }, _('Available') + ' / ' + _('Size'))
+			])
+		]);
+
+		cbi_update_table(table, mounts.filter( function(mount) {
+				return mount.device.match('/dev/sd*') || 
+					mount.mount.match('/overlay') 
```

Comment:
On my system I have the following output for `ubus call luci getMountPoints`
```
{
        "result": [
                {
                        "avail": 0,
                        "device": "/dev/root",
                        "free": 0,
                        "mount": "/rom",
                        "size": 21495808
                },
                {
                        "avail": 61038592,
                        "device": "tmpfs",
                        "free": 61038592,
                        "mount": "/tmp",
                        "size": 62152704
                },
                {
                        "avail": 75231232,
                        "device": "/dev/ubi0_3",
                        "free": 79314944,
                        "mount": "/overlay",
                        "size": 80576512
                },
                {
                        "avail": 75231232,
                        "device": "overlayfs:/overlay",
                        "free": 79314944,
                        "mount": "/",
                        "size": 80576512
                },
                {
                        "avail": 13180928,
                        "device": "/dev/ubi0_0",
                        "free": 13918208,
                        "mount": "/permdata",
                        "size": 13955072
                },
                {
                        "avail": 524288,
                        "device": "tmpfs",
                        "free": 524288,
                        "mount": "/dev",
                        "size": 524288
                }
        ]
}
```
I would only filter for `mount.device` and not also for `mount.mount` if `mount.device` does not match. With your filter my device `/dev/ubi0_0` does not match and so will not get shown.
Interesting matches from my point of view would be:
`/dev/ubi*`
`overlayfs:/overlay`
`/dev/sd*`

Reply:
I didn't include overlayfs:/overlay as it was redundant to have 2 row with the same values. Am I wrong?

Reply:
You're right about that. I'm not sure which is better either.
I would just show everything and not add any special treatment.
At some poin in the feature something will change again and then you have to look at why it is not being displayed now. The user has to know what he is seeing and how to interpret the data.

I would report everything. If we adopt this, then my mount points will not be displayed.

Reply:
Was thinkg of displaying only ubi and sd* mount. What do you think? Or display a copy of the table in the mount page?

I mean after all we receive all the mount so I think you are right and we should display all of them unconditionally.

Reply:
Let's start like this. Can you please add the acl? And change the var (see above)?

Reply:
Sorry you mean display all or display sda and ubi ?